feature/qos/otg_tests/ingress_police_nhg/README.md

```diff
@@ -0,0 +1,216 @@
+# TE-18.2 QoS scheduler with 1 rate 2 color policer, classifying on next-hop group
+
+## Summary
+
+Use the gRIBI applied ip entries from TE-18.1 gRIBI. Configure an ingress scheduler
+to police traffic using a 1 rate, 2 color policer. Configure a classifier to match
+traffic on a next-hop-group.  Apply the configuration to a VLAN on an aggregate
+interface.  Send traffic to validate the policer.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+Use TE-18.1 test environment setup.
+
+## Procedure
+
+### TE-18.3.1 Generate and push configuration
+
+* Generate config for 2 scheduler polices with an input rate limit.  
+* Generate config for 2 classifiers which match on next-hop-group.
+* Generate config for 2 input policies which map the scheduler and classifers
+  together.
+* Generate config to apply classifer and scheduler to DUT subinterface with vlan.
+* Use gnmi.Replace to push the config to the DUT.
+
+```yaml
+---
+openconfig-qos:
+  scheduler-policies:
+    - scheduler-policy: "limit_2Gb"
+      config:
+        name: "limit_2Gb"
+      schedulers:
+        - scheduler: 0
+          config:
+              type: ONE_RATE_TWO_COLOR
+              sequence: 0
+          one-rate-two-color:
+            config:
+              cir: 2000000000           # 2Gbit/sec
+              bc: 100000                 # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+
+    - scheduler-policy: "limit_1Gb"
+      config:
+        name: "limit_1Gb"
+      schedulers:
+        - scheduler: 0
+          config:
+              type: ONE_RATE_TWO_COLOR
+              sequence: 0
+          one-rate-two-color:
+            config:
+              cir: 1000000000           # 1Gbit/sec
+              bc: 100000                # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+  classifers:
+    - classifer: “dest_A”
+      config:
+        name: “dest_A”
+      terms:
+        - term:   # repeated for address in destination A
+          config:
+            id: "match_1_dest_A"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_A"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
```

Comment:
Here and below, this is matching on next-hop-group.  It should be changed to match on destination MAC.